.travis.yml

```diff
@@ -1,35 +1,62 @@
 os: linux
 dist: focal
-language: c++
+language: minimal
+
+services:
+  - docker
+
 env:
-  global:
-       - DEBIAN_FRONTEND: noninteractive
-addons:
-  apt:
-    packages:
-        - wget
-        - locales
-        - python3-pip
+  # Testing QuickLogic toolchain on all OSes
+  - TOOLCHAIN=quicklogic      OS=ubuntu OS_VERSION=trusty
+  - TOOLCHAIN=quicklogic      OS=ubuntu OS_VERSION=xenial
+  - TOOLCHAIN=quicklogic      OS=ubuntu OS_VERSION=bionic
+  - TOOLCHAIN=quicklogic      OS=ubuntu OS_VERSION=eoan
+  - TOOLCHAIN=quicklogic      OS=ubuntu OS_VERSION=focal
+  - TOOLCHAIN=quicklogic      OS=centos OS_VERSION=6
+  - TOOLCHAIN=quicklogic      OS=centos OS_VERSION=7
+  - TOOLCHAIN=quicklogic      OS=centos OS_VERSION=8
+  # Testing Xilinx 7 Series toolchain on all OSes
+  - TOOLCHAIN=xilinx-series-7 OS=ubuntu OS_VERSION=trusty
+  - TOOLCHAIN=xilinx-series-7 OS=ubuntu OS_VERSION=xenial
+  - TOOLCHAIN=xilinx-series-7 OS=ubuntu OS_VERSION=bionic
+  - TOOLCHAIN=xilinx-series-7 OS=ubuntu OS_VERSION=eoan
+  - TOOLCHAIN=xilinx-series-7 OS=ubuntu OS_VERSION=focal
+  - TOOLCHAIN=xilinx-series-7 OS=centos OS_VERSION=6
+  - TOOLCHAIN=xilinx-series-7 OS=centos OS_VERSION=7
+  - TOOLCHAIN=xilinx-series-7 OS=centos OS_VERSION=8
 
 before_install:
         - sudo locale-gen "en_US.UTF-8"
         - export LANG="en_US.UTF-8"
         - source .github/travis/common.sh
         - rm README.rst && make README.rst && .github/travis/git-check.sh
-        - pip install git+https://github.com/antmicro/tuttest
-        - tuttest README.rst unnamed0 | bash -
-jobs:
-  include:
-    - stage: SymbiFlow
-      install:
-            - tuttest xc7/README.rst unnamed0 | bash -
+        - pip install git+https://github.com/antmicro/tuttest@prefix-lines-with#egg=tuttest
 
-      script:
-            - tuttest xc7/README.rst unnamed1,unnamed2,unnamed3,unnamed4,unnamed5 | bash -
-    - #
-      install:
-            - tuttest eos-s3/README.rst unnamed0 | bash -
+install:
```

Comment:
These probably should be part of the documentation (and installed using tuttest)?

Reply:
I added required ``wget`` package to README.rst.